gss-genr.c

```diff
@@ -84,6 +84,24 @@ ssh_gssapi_get_buffer_desc(struct sshbuf *b, gss_buffer_desc *g)
 	return 0;
 }
 
+/* sshpkt_get of gss_bufer_desc
+ * XXX similar as above, but because of opaque ssh structure,
+ * I can not reuse the code
```

Comment:
This makes sense to mention in a commit message, but I don't think it really needs to sit around as an XXX comment, because it isn't something that can reasonably be fixed in the GSSAPI key exchange patch, but rather a consequence of OpenSSH's layout.

(The comment is incorrect as it stands, incidentally, and if you decide to keep it then you should fix that.  `struct ssh` isn't opaque - it's defined in packet.h, and gss-genr.c could include that if it wanted to.  However, `struct session_state` *is* opaque, being defined only in packet.c.)

Reply:
True. I dropped the comment and updated the commit message here.